packages/javascript-sdk/src/token-storage/index.ts

```diff
@@ -71,24 +76,25 @@ abstract class TokenStorage {
       return await SessionStorageWrapper.remove(clientId);
     } else if (tokenStore === 'localStorage') {
       return await LocalStorageWrapper.remove(clientId);
+
+      // Preserving this condition for communicating its removal
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
     } else if (tokenStore === 'indexedDB') {
-      return await IndexedDBWrapper.remove(clientId);
+      console.warn('IndexedDB is not supported in this version.');
     } else if (tokenStore && tokenStore.remove) {
       // User supplied token store
       return await tokenStore.remove(clientId);
     }
-    // if tokenStore is undefined, default to localStorage
     return await LocalStorageWrapper.remove(clientId);
   }
 
   private static getClientConfig(): {
     clientId: string;
-    tokenStore: TokenStoreObject | 'indexedDB' | 'sessionStorage' | 'localStorage' | undefined;
+    tokenStore: TokenStoreObject | 'sessionStorage' | 'localStorage' | undefined;
```

Comment:
Removed the `indexedDB` type and also destructured with defaults to clean the code up a bit.